docs/en/observability/logs-threshold-alert.asciidoc

```diff
@@ -18,8 +21,58 @@ The comparators available for conditions depend on the chosen field. The combina
 There are several key supported use cases. You can create alerts based on fields containing or matching a text pattern,
 alerts based on a numeric field and arithmetic operator, or a single alert with multiple conditions.
 
+Each of these comparators is backed by a different Elasticsearch query type, and in some scenarios it is important to know what these are so that you can configure your alert correctly. The comparators listed above map to the following Elasticsearch query types:
```

Comment:
```suggestion
A different {es} query type backs each of these comparators, and in some scenarios, it is important to know what these are so that you can configure your alert correctly. The comparators listed above map to the following {es} query types:
```